caddyconfig/caddyfile/formatter.go

```diff
@@ -0,0 +1,141 @@
+// Copyright 2015 Matthew Holt and The Caddy Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package caddyfile
+
+import (
+	"bytes"
+	"io"
+	"unicode"
+)
+
+// Formatter is a type that implements caddyconfig.Formatter.
+// This is used to format a Caddyfile.
+type Formatter struct {
+}
+
+// Format formats the body of Caddyfile.
+func (f Formatter) Format(body []byte) []byte {
```

Comment:
This method can just be a function!
```suggestion
// Format formats a Caddyfile to conventional standards.
func Format(body []byte) []byte {
```